DE/asciidoc/src/10_quality_scenarios.adoc

```diff
@@ -58,11 +58,11 @@ System in bestimmten Situationen geschieht.
 
 Wesentlich für die meisten Softwarearchitekten sind zwei Arten von Szenarien:
 
-*  Nutzungsszenarien (auch genannt Anwendungs- oder Anwendungsfallszenarien) beschreiben, wie das System zur Laufzeit auf einen
+* Nutzungsszenarien (auch genannt Anwendungs- oder Anwendungsfallszenarien) beschreiben, wie das System zur Laufzeit auf einen
 bestimmten Auslöser reagieren soll. Hierunter fallen auch Szenarien zur Beschreibung von Effizienz oder Performance.
 Beispiel: Das System beantwortet eine Benutzeranfrage innerhalb einer Sekunde.
-*  Änderungsszenarien beschreiben eine Modifikation des Systems oder seiner unmittelbarer Umgebung. Beispiel: Eine zusätzliche
- Funktionalität wird implementiert oder die Anforderung an ein Qualitätsmerkmal ändert sich.
+* Änderungsszenarien beschreiben eine Modifikation des Systems oder seiner unmittelbarer Umgebung. Beispiel: Eine zusätzliche
```

Comment:
> ... seiner unmittelbarer Umgebung ...

unmittelbarer -> unmittelbaren

Das Beispiel würde ich mit einem harten Umbruch in eine neue Zeile verschieben.